spec/std/log/context_spec.cr

```diff
@@ -0,0 +1,119 @@
+require "spec"
+require "log"
+
+def c(value)
```

Comment:
These helper methods should be private to the spec file to avoid global pollution.